server/babel.config.js

```diff
@@ -0,0 +1,7 @@
+module.exports = {
```

Comment:
do we need babel for backend?

Reply:
Unfortunately, Jest requires babel when using TypeScript [here](https://jestjs.io/docs/getting-started#using-typescript)